lib/patterns.rb

```diff
@@ -92,8 +92,6 @@ module Patterns
     sidekiq-pro
     graphql-pro
 
-    # Blacklisted internal Rails dependency misspellings
```

Comment:
It will be nice to leave this comment in there even reworded.

Reply:
We can have commnet outside `%w[..]`, which would feel a bit misplaced. Perhaps, following would be more suited?
```
 # blacklists ruby core libraries, internal rails dependencies and their misspellings 
 GEM_NAME_BLACKLIST    = %w[...
```